python/rmm/rmm/pylibrmm/memory_resource.pyx

```diff
@@ -991,7 +1032,7 @@ cpdef void _initialize(
     try:
         original_device = getDevice()
     except CUDARuntimeError as e:
-        if e.status == cudaError_t.cudaErrorNoDevice:
+        if e.status == runtime.cudaError_t.cudaErrorNoDevice:
```

Comment:
Why?

Reply:
We adopted this naming pattern (`from cuda.bindings import runtime` and `runtime.*`, etc.) in several RAPIDS libraries when cuda-python changed its layout. This keeps us from having to import a bunch of specific names from `cuda.bindings.runtime`, and it makes it clear where they come from. In this particular file we have both `cyruntime` and `runtime` and I want it to be clear which names are from which modules.